internal/k8shandler/forwarding.go

```diff
@@ -71,3 +80,21 @@ func (clusterRequest *ClusterLoggingRequest) SetOutputSecrets() {
 		clusterRequest.OutputSecrets[output.Name] = secret
 	}
 }
+
+func (clusterRequest *ClusterLoggingRequest) getLogCollectorServiceAccountTokenSecret() (*corev1.Secret, error) {
```

Comment:
Can we use similar logic in the existing SetOutputSecrets function to just get the secret?  Do we need additional verification?  I believe there is an established pattern for SA token secrets upon which we can rely and may not require additional validation